### DISCUSSION TITLE
Role Suggestion

### DISCUSSION THREAD

**Discussion by lumbar527** [Dungeons-of-Kathallion/Dungeons-of-Kathallion#28] [Roles]
This is the discussion to suggest we give organization permissions to people.

**Steps**
1. Suggest someone; explain why you think they would be a good addition
2. The suggested person replies.
3. The suggested person reads the [Contributing guidelines](https://github.com/Dungeons-of-Kathallion/Dungeons-of-Kathallion/blob/main/.github/CONTRIBUTING.md)
4. Admins decide if they should be given permissions.

**Comment by lumbar527** [comment]
I thing @OcelotWalrus should be added to the `coder` team.

**Reply by OcelotWalrus** [reply]
Sure! Just a question, why is the purpose of this github 'organization'? Do you plan on making other games?

**Reply by lumbar527** [reply]
It's an organization for more permissions on it, and to organize repos. I'm planning on having more things related to Dungeons-of-Kathallion here.

Oh, and have you read the newest version of the contributing guidelines?

**Reply by lumbar527** [reply]
No plan or anything, but one this game is finished, there could be a sequel, too.

**Reply by OcelotWalrus** [reply]
> Oh, and have you read the newest version of the contributing guidelines?

Not yet.

**Reply by lumbar527** [reply]
> > Oh, and have you read the newest version of the contributing guidelines?
> 
> Not yet.

Tell me when you do.

**Reply by OcelotWalrus** [reply]
I finished by now.

**Reply by lumbar527** [reply]
I've invited you.

**Reply by OcelotWalrus** [reply]
Okay, thanks!
